src/modal/parts/Container.jsx

```diff
@@ -24,6 +37,35 @@ const Modal = ({ children }) => {
         }
     }, [transitionState]);
 
+    const countryModalContainer = () => {
+        switch (country) {
+            case 'GB':
+                return (
+                    <div className="modal__content-wrapper" ref={contentWrapper}>
+                        <div className="modal__content-background">
+                            <Header />
+                            <div className="modal__content">
+                                <main className="modal__main">{children}</main>
+                            </div>
+                        </div>
+                    </div>
+                );
+            default:
```

Comment:
Consider including `case 'US':` and `case 'DE':` above `default:` as well to be explicit about the desired countries this container is for. Consider immediately invoking the function and just storing the value.